lib/core/presentation/pages/event/event_location_setting_page/sub_pages/event_location_setting_detail_page.dart

```diff
@@ -0,0 +1,295 @@
+import 'package:app/core/application/event/event_location_setting_bloc/event_location_setting_bloc.dart';
+import 'package:app/core/config.dart';
+import 'package:app/core/domain/common/entities/common.dart';
+import 'package:app/core/presentation/widgets/common/appbar/lemon_appbar_widget.dart';
+import 'package:app/core/presentation/widgets/common/button/linear_gradient_button_widget.dart';
+import 'package:app/core/presentation/widgets/lemon_text_field.dart';
+import 'package:app/core/utils/snackbar_utils.dart';
+import 'package:app/i18n/i18n.g.dart';
+import 'package:app/theme/spacing.dart';
+import 'package:app/theme/typo.dart';
+import 'package:auto_route/auto_route.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:flutter_screenutil/flutter_screenutil.dart';
+import 'package:flutter_google_places_hoc081098/flutter_google_places_hoc081098.dart';
+import 'package:flutter_vibrate/flutter_vibrate.dart';
+import 'package:formz/formz.dart';
+import 'package:google_maps_webservice/places.dart';
+import 'package:google_api_headers/google_api_headers.dart';
+
+@RoutePage()
+class EventLocationSettingDetailPage extends StatefulWidget {
+  const EventLocationSettingDetailPage({Key? key, this.address})
+      : super(key: key);
+
+  final Address? address;
+
+  @override
+  State<EventLocationSettingDetailPage> createState() =>
+      _EventLocationSettingDetailPageState();
+}
+
+class _EventLocationSettingDetailPageState
+    extends State<EventLocationSettingDetailPage> {
+  final TextEditingController placeDetailsController = TextEditingController();
+  final TextEditingController titleController = TextEditingController();
+  final TextEditingController street1Controller = TextEditingController();
+  final TextEditingController street2Controller = TextEditingController();
+  final TextEditingController cityController = TextEditingController();
+  final TextEditingController regionController = TextEditingController();
+  final TextEditingController postalController = TextEditingController();
+  final TextEditingController countryController = TextEditingController();
+
+  @override
+  void initState() {
+    super.initState();
+    WidgetsBinding.instance.addPostFrameCallback((timeStamp) {
+      context
+          .read<EventLocationSettingBloc>()
+          .add(EventLocationSettingEvent.init(address: widget.address));
+      if (widget.address != null) {
+        titleController.text = widget.address?.title ?? '';
+        street1Controller.text = widget.address?.street1 ?? '';
+        street2Controller.text = widget.address?.street2 ?? '';
+        cityController.text = widget.address?.city ?? '';
+        regionController.text = widget.address?.region ?? '';
+        postalController.text = widget.address?.postal ?? '';
+        countryController.text = widget.address?.country ?? '';
+      }
+    });
+  }
+
+  @override
+  void dispose() {
+    placeDetailsController.clear();
```

Comment:
we dont have to clear when widget dispose, not calling this will not create any side-effect, so it's ok to remove these